Bio/SeqIO/Interfaces.py

```diff
@@ -91,6 +94,10 @@ def __iter__(self):
         """
         return self
 
+    @abstractmethod
+    def parse(self, handle: IO[Any]) -> Iterator[SeqRecord]:
```

Comment:
Can you take out the annotation for now please.

Reply:
Done